torchmultimodal/models/albef.py

```diff
@@ -211,10 +211,10 @@ def forward(
         image_embeds_neg, text_embeds_neg, text_atts_neg = self._neg_embeddings(
             outputs.image_embeddings, outputs.text_embeddings, text_atts, similarity
         )
-        multimodal_embeddings_neg = self.model.multimodal_encoder(
-            torch.cat([image_embeds_neg, outputs.image_embeddings], dim=0),
+        multimodal_embeddings_neg = self.albef_model.multimodal_encoder(
             torch.cat([outputs.text_embeddings, text_embeds_neg], dim=0),
             torch.cat([text_atts, text_atts_neg], dim=0),
+            torch.cat([image_embeds_neg, outputs.image_embeddings], dim=0),
```

Comment:
This is just a continuation of the changes you made to multimodal encoder in #224, right?